src/RadixDlt.NetworkGateway.PostgresIntegration/DbQueryExtensions.cs

```diff
@@ -174,7 +174,8 @@ public static IQueryable<LedgerTransaction> GetLatestLedgerTransactionAtEpochRou
         return dbContext
             .LedgerTransactions
             .Where(lt => lt.Epoch == epoch && lt.RoundInEpoch >= round && lt.IndexInRound == 0)
-            .OrderByDescending(lt => lt.StateVersion)
+            .OrderByDescending(lt => lt.Epoch)
```

Comment:
Hmm I don't think this query is actually correct.

It says it "Returns most recently committed ledger transaction at or before given epoch and round." but then it filters to "lt => lt.Epoch == epoch && lt.RoundInEpoch >= round && lt.IndexInRound == 0".

Perhaps we should change the docs to "Returns the most recently committed round change ledger transaction at or before given epoch and round." and then change the where to just be on `lt.IndexInRound` ?

Reply:
Okay, we've agreed that we probably want to change this at some point (likely to require an exact match) but for now, we'll ignore this logic and just fix the issue.

In which case - because we have a "where lt.Epoch == epoch" exact match, we don't need to order by it. We probably just want to order by descending on the round?

Or taking another step back, this should just equate to a direct look up of epoch and round?? So maybe we just do that?

Reply:
Decided to go with exact epoch+round lookup.